examples/FactSet.AnalyticsAPI.Engines.Example/Examples/PAEngineMultipleUnitLinkedTemplatedComponentExample.cs

```diff
@@ -0,0 +1,278 @@
+                        {
+                            var maxAge = getStatusResponse.Headers["Cache-Control"][0];
+                            if (string.IsNullOrWhiteSpace(maxAge))
+                            {
+                                Console.WriteLine("Sleeping for 2 seconds");
+                                // Sleep for at least 2 seconds.
+                                Thread.Sleep(2000);
+                            }
+                            else
+                            {
+                                var age = int.Parse(maxAge.Replace("max-age=", ""));
+                                Console.WriteLine($"Sleeping for {age} seconds");
+                                Thread.Sleep(age * 1000);
+                            }
+                        }
+                    }
+
+                    getStatusResponse = calculationApi.GetCalculationStatusByIdWithHttpInfo(id: calculationId);
+                    status = getStatusResponse.Data;
+                }
+                Console.WriteLine("Calculation Completed");
+
+                
+                foreach (var calculation in status.Data.Units)
+                {
+                    if (calculation.Value.Status == CalculationUnitStatus.StatusEnum.Success)
+                    {
+                        var resultResponse = calculationApi.GetCalculationUnitResultByIdWithHttpInfo(id: calculationId, unitId: calculation.Key);
+                        PrintResult(result: resultResponse.Data);
+                    }
+                    else
+                    {
+                        Console.WriteLine($"Calculation Unit Id : {calculation.Key} Failed!!!");
+                        Console.WriteLine($"Error message : {calculation.Value.Errors?.FirstOrDefault()?.Detail}");
+                    }
+                }
+
+                Console.ReadKey();
+            }
+            catch (ApiException e)
+            {
+                Console.WriteLine($"Status Code: {e.ErrorCode}");
+                Console.WriteLine($"Reason : {e.Message}");
+                Console.WriteLine(e.StackTrace);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                throw;
+            }
+        }
+
+        private static Configuration GetApiConfiguration()
+        {
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
+            if (_engineApiConfiguration != null)
+            {
+                return _engineApiConfiguration;
+            }
+
+            _engineApiConfiguration = new Configuration
+            {
+                BasePath = BasePath,
+                Username = UserName,
+                Password = Password
+            };
+            
+            // Uncomment below lines for adding the proxy configuration
+            //System.Net.WebProxy webProxy = new System.Net.WebProxy("http://myProxyUrl:80/");
```

Comment:
Set Proxy URL as environment variable and read proxy URL also from environment variable.

Reply:
done https://github.com/factset/analyticsapi-engines-dotnet-sdk/pull/68/commits/abcdd14a92a497c5aa6684f882b5a18fe5ca6004